xml/art_virtualization-best-practices.xml

```diff
@@ -3241,6 +3241,162 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
       </para>
 <screen>&prompt.root;<command>echo 1 > /sys/devices/system/cpu/cpu<replaceable>X</replaceable>/online</command></screen>
     </sect2>
+
+    <sect2 xml:id="sec-vm-guest-scsi-pr">
+      <title>SCSI3 persistent reservation (S3-PR) on a multipathed device</title>
+      <para>
+        SCSI persistent reservations allow restricting access to block devices
+        to specific initiators in a shared storage setup.  When implementing
+        clustering of &vmguest;, it is a common requirement for &vmguest; to
```

Comment:
What is "clustering of vmguest"? Maybe "clustering of vmguests"? or "clustering of the vmguest"?

Reply:
OK, use "clustering of vmguests" instead.

Reply:
ok, do we have a reference to doc where we explain what VM clustering of guest is?